ci/jenkins/Jenkins_steps.groovy

```diff
@@ -521,6 +521,19 @@ def compile_windows_gpu_mkldnn() {
     }]
 }
 
+def test_static_scala_cpu() {
+  return ['Static build CPU 14.04 Scala' : {
+    node(NODE_LINUX_CPU) {
+        ws('workspace/ut-publish-scala-cpu') {
+          withEnv(["MAVEN_PUBLISH_OS_TYPE=linux-x86_64-cpu", "mxnet_variant=mkl"]) {
```

Comment:
Please set environment variables in the runtime functions (see other thread for background)